src/cgalutils-polyhedron.cc

```diff
@@ -270,7 +270,7 @@ namespace CGALUtils {
 	{
 		bool err = false;
 		typedef typename Polyhedron::Vertex                                 Vertex;
-		typedef typename Polyhedron::Vertex_const_iterator                  VCI;
+		//typedef typename Polyhedron::Vertex_const_iterator                  VCI;
```

Comment:
If it's not used, we should just kill it

Reply:
done :)